go-kit/metrics/provider/librato/batcher.go

```diff
@@ -196,9 +161,9 @@ type measurement struct {
 	Attributes map[string]interface{} `json:"attributes,omitempty"`
 	Tags       map[string]string      `json:"tags"`
 
+	Count  int64   `json:"count"`
```

Comment:
This rearrangement seems unnecessary, but I'm going to guess that it's sole purpose is to make the JSON marshaling align correctly?

Reply:
Yep :)

Reply:
Also it's an aesthetic in terms of int64 being separated from the float64s.